site/docs/4.5/extend/icons.md

```diff
@@ -39,3 +39,4 @@ While we haven't tried these out, they do look promising and provide multiple fo
 - [Dripicons](http://demo.amitjakhu.com/dripicons/)
 - [Ikons](http://ikons.piotrkwiatkowski.co.uk/)
 - [Icons8](https://icons8.com/)
+- [CoreUI Icons](https://icons.coreui.io/)
```

Comment:
@mdo it seems we are missing another one from v5, `https://icofont.com/`, you want me to backport that too?